### PR TITLE
Drop dependency on Net::Interface

### PR DESCRIPTION
**Pull request by mattias-p**
Results in a less helpful error message in a rare situation - i.e. when
--sourceaddr is specified with an unreasonable value.

## Purpose

Makes installation easier/possible on CentOS and Alpine.

## Context

This makes it easier to implement #216.

## Changes

This PR removes some code to display a more helpful error message in an edge case. To compensate the documentation is improved for the involved CLI parameter.

The dependency on Net::Interface is removed.

## How to test this PR

Make sure the usage documentation for the --sourceaddr option clarifies that you need to specify an IP address that is configured on a network interface:
```sh
zonemaster-cli --help
```

Make sure we don't depend on Net::Interface:
```sh
grep Net::Interface Makefile.PL
```

### CODE REVIEW

File: lib/Zonemaster/CLI.pm [hunk 1]
```diff
@@ -271,7 +270,10 @@ has 'sourceaddr' => (
     is            => 'ro',
     isa           => 'Str',
     required      => 0,
-    documentation => __( 'Local IP address that the test engine should try to send its requests from.' ),
+    documentation => __(
+            'Source IP address used to send queries. '
+          . 'Setting an IP address not correctly configured on a local network interface causes cryptic error messages.'
```
[the remark this answers is not in the document]
Comment (mattias-p) [reply lib/Zonemaster/CLI.pm:275]: I don't see when that would be useful. But I'll add sourceaddr to the POD.

[the remark this answers is not in the document]
Comment (matsduf) [reply lib/Zonemaster/CLI.pm:275]: > I don't see when that would be useful.

The proposed text is to further prevent users from using the option. It is really for special cases.
